packages/wrangler/src/deployments.ts

```diff
@@ -36,6 +36,12 @@ export async function deployments(
 	accountId: string,
 	scriptName: string | undefined
 ) {
+	if (!scriptName) {
+		throw new Error(
+			"Required Worker name missing. Please specify the Worker name in wrangler.toml, or pass it as an argument with `--name`"
```

Comment:
I don't see a test that actually checks that this message is displayed when a worker name is not provided.

Reply:
Can tests be added to check this for all three conditions? `--name`, `wrangler.toml`, and no name provided?